src/backend/transliterator/index.ts

```diff
@@ -110,26 +122,128 @@ export class Transliterator extends Construct {
       environment.CODE_ARTIFACT_REPOSITORY_ENDPOINT = props.codeArtifact.repositoryNpmEndpoint;
     }
 
-    const lambda = new Handler(this, 'Resource', {
-      description: `Creates ${props.language} documentation from jsii-enabled npm packages`,
+    this.logGroup = new LogGroup(this, 'LogGroup', { retention: props.logRetention });
+    this.containerDefinition = new Container(this, 'Resource', {
       environment,
-      logRetention: props.logRetention ?? RetentionDays.TEN_YEARS,
-      memorySize: 10_240, // Currently the maximum possible setting
-      timeout: Duration.minutes(15),
-      tracing: Tracing.PASS_THROUGH,
-      vpc: props.vpc,
-      vpcSubnets: props.vpcSubnets,
+      logging: LogDrivers.awsLogs({ logGroup: this.logGroup, streamPrefix: 'transliterator' }),
+      taskDefinition: new FargateTaskDefinition(this, 'TaskDefinition', {
+        cpu: 4_096,
+        memoryLimitMiB: 8_192,
+      }),
     });
-    this.function = lambda;
 
-    repository?.grantReadFromRepository(this.function);
+    repository?.grantReadFromRepository(this.taskDefinition.taskRole);
 
     // The handler reads & writes to this bucket.
```

Comment:
nit: minor clarification
```suggestion
    // The task handler reads & writes to this bucket.
```